cocos/platform/desktop/CCGLViewImpl-desktop.cpp

```diff
@@ -911,6 +911,10 @@ void GLViewImpl::onGLFWframebuffersize(GLFWwindow* window, int w, int h)
         _retinaFactor = 1;
         glfwSetWindowSize(window, static_cast<int>(frameSizeW * _retinaFactor * _frameZoomFactor), static_cast<int>(frameSizeH * _retinaFactor * _frameZoomFactor));
     }
+
+    //FIXME:https://github.com/cocos2d/cocos2d-x/issues/16779
+    Director::getInstance()->setViewport();
+    Director::getInstance()->mainLoop();
```

Comment:
Why should invoke mainLoop after setViewpoint?, mainLoop will be automatically called, right?